src/utils/Cargo.toml

```diff
@@ -17,7 +17,7 @@ displaydoc = "0.2.4"
 versionize = "0.1.10"
```

Comment:
on the commit message: biggy-back -> piggy-back

Reply:
Fixed